packages/vhd-lib/tests/utils.js

```diff
@@ -21,7 +21,7 @@ async function createRandomFile(name, sizeMB) {
 exports.createRandomFile = createRandomFile
 
 async function checkFile(vhdName) {
-  await execa('vhd-util', ['check', '-p', '-b', '-t', '-n', vhdName])
+  await execa('qemu-img', ['convert', vhdName, 'outputFile.qcow2'])
```

Comment:
Does it work well? Is it able to detect a broken/invalid VHD?

Does it left unused files after the test has run?

We should add a comment to explain what we're trying to do here.

Reply:
The test was successful so I guess it worked as expected. I don't think there is a test for broken vhd file. Maybe I could try (even though I don't know enough about vhd file to create a broken one to implement the test but I'll something).

All files are removed after each test.

I'll add a comment.

Reply:
1. I understand it's not easy to test, unfortunately this line is completely useless if it does not handle this properly.
2. Perfect.
3. Ok.